api/v1alpha1/etcdcluster_types.go

```diff
@@ -66,6 +70,16 @@ type EtcdCluster struct {
 	Status EtcdClusterStatus `json:"status,omitempty"`
 }
 
+// calculateQuorumSize returns minimum quorum size for current number of replicas
+func (r *EtcdCluster) calculateQuorumSize() int {
+	replicas := *r.Spec.Replicas
+	if replicas%2 == 0 {
```

Comment:
I am not sure that you changed it correctly.
6 replicas
6-1=5
5/2=2.5
Ceil(2.5)=3

And it should be 4.

If you change "replicas - 1" to "replicas + 1" it will be correct I think.

Reply:
Thank you!